elytron/src/main/java/org/wildfly/extension/elytron/ModifiableRealmDecorator.java

```diff
@@ -414,6 +415,30 @@ protected void executeRuntimeStep(final OperationContext context, final ModelNod
                     .build();
         }
 
+        static class ScramDigest {
+            static final SimpleAttributeDefinition ALGORITHM = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.ALGORITHM, ModelType.STRING)
+                    .setRequired(false)
+                    .setDefaultValue(new ModelNode(ScramDigestPassword.ALGORITHM_SCRAM_SHA_1))
```

Comment:
It would be good to use a stronger hash function by default. It looks like SHA-512 is being used for the defaults for other password types.